react-events/my-app/src/CustomButton.js

```diff
@@ -0,0 +1,12 @@
+import './CustomButton.css';
+
+export default function CustomButton({ text, className, message }) {
+  function handleClick() {
```

Comment:
This is correct for the first phase of the exercise. Now do the steps near the end that have you move this function into App.